apps/src/lab2/types.ts

```diff
@@ -173,6 +173,8 @@ export interface LevelProperties {
   teacherMarkdown?: string;
   predictSettings?: LevelPredictSettings;
   submittable?: boolean;
+  finishUrl?: string;
+  finishDialog?: string;
```

Comment:
Not blocking, but we could create a shared constant in `shared_constants.rb` like `FinishDialogType` or similar, and then use here for some automatic typescript compile-time checking.

Reply:
That's a good point.  I figured this would evolve relatively soon as we learn more about finish experiences here and elsewhere.